config/initializers/application_record_serializing.rb

```diff
@@ -0,0 +1,17 @@
+ar_serializing = Module.new do
+  def as_json(arg = {})
+    previous = super
+    return previous unless arg.key?(:prefixes)
+    /\/v(?<major>\d+)[x\.]?(?<minor>\d+)?\// =~ arg[:prefixes].first
+    version = [major, minor].compact.join(".")
+    schema  = Api::Docs[version].definitions[self.class.name]
```

Comment:
Is it possible that we might not have a definition in the yaml. Should we raise that as an error

Reply:
Yeah, we should probably do `return previous unless schema`.  Though that might be a workaround we don't want to do.  @bdunne?

Reply:
I would expect some sort of error if you're trying to serialize something (to return to the client) that isn't defined in the swagger file.

Reply:
@bdunne I think it's the other way around.  They have it in the swagger file, but it doesn't have a corresponding model.